mcda/utils/utils_for_main.py

```diff
@@ -47,36 +47,41 @@ def check_config_error(condition: bool, error_message: str):
 
 def check_config_setting(condition_robustness_on_weights: bool,
                          condition_robustness_on_indicators: bool,
-                         mc_runs: int) -> (int, int):
+                         mc_runs: int, random_seed: int) -> (int, int):
     """
-        Checks configuration settings and logs information messages.
+    Checks configuration settings and logs information messages.
 
-        Parameters:
-        - condition (bool): The condition to check.
+    Parameters:
+    - is_robustness_weights, is_robustness_indicators, booleans indicating if robustness is considered
+    on weights or indicators.
 
-        Returns:
-        :rtype: Tuple[int, int]
-        :return: is_robustness_weights, is_robustness_indicators, a tuple containing flags indicating robustness
-                 on weights and indicators.
+    Example:
+    ```python
+    is_robustness_weights, is_robustness_indicators = check_config_setting(True, False, 1000, 42)
+    ```
 
-        Example:
-        ```python
-        is_robustness_weights, is_robustness_indicators = check_config_setting(True, "False,
-                                                                               "True", "False", 1000)
-        ```
-        """
+    :rtype: Tuple[int, int]
```

Comment:
Here this list is again confusing with the parameter declaration above.

Reply:
Do you understand my logic now? Don't you like it?
if needed, a short comment is added about parameters and returns, and a full list of parameters and returns is always at the bottom of each docstring.

This produces a function description like this (if you hover the mouse over its name):

<img width="529" alt="Screenshot 2024-03-13 at 15 22 55" src="https://github.com/wetransform-os/ProMCDA/assets/26147427/d318a6aa-6213-43a3-b7a0-816a9f23b8b7">

Reply:
Cool. Thanks for the explanation.